vision/google/cloud/vision_v1/types.py

```diff
@@ -15,46 +15,40 @@
 from __future__ import absolute_import
 import sys
 
+from google.api_core.protobuf_helpers import get_messages
+
 from google.api import http_pb2
+from google.cloud.vision_v1.proto import geometry_pb2
+from google.cloud.vision_v1.proto import image_annotator_pb2
+from google.cloud.vision_v1.proto import text_annotation_pb2
+from google.cloud.vision_v1.proto import web_detection_pb2
+from google.longrunning import operations_pb2
 from google.protobuf import any_pb2
 from google.protobuf import descriptor_pb2
+from google.protobuf import empty_pb2
+from google.protobuf import timestamp_pb2
 from google.protobuf import wrappers_pb2
 from google.rpc import status_pb2
 from google.type import color_pb2
 from google.type import latlng_pb2
 
-from google.api_core.protobuf_helpers import get_messages
-from google.cloud.vision_v1.proto import geometry_pb2
-from google.cloud.vision_v1.proto import image_annotator_pb2
-from google.cloud.vision_v1.proto import text_annotation_pb2
-from google.cloud.vision_v1.proto import web_detection_pb2
-
-
-_shared_modules = [
-    http_pb2,
-    any_pb2,
-    descriptor_pb2,
-    wrappers_pb2,
-    status_pb2,
-    color_pb2,
-    latlng_pb2,
-]
-
-_local_modules = [
-    geometry_pb2,
-    image_annotator_pb2,
-    text_annotation_pb2,
-    web_detection_pb2,
-]
-
 names = []
-
-for module in _shared_modules:
-    for name, message in get_messages(module).items():
-        setattr(sys.modules[__name__], name, message)
-        names.append(name)
-
-for module in _local_modules:
+for module in (
+        http_pb2,
+        geometry_pb2,
+        image_annotator_pb2,
+        text_annotation_pb2,
+        web_detection_pb2,
+        operations_pb2,
+        any_pb2,
+        descriptor_pb2,
+        empty_pb2,
+        timestamp_pb2,
+        wrappers_pb2,
+        status_pb2,
+        color_pb2,
+        latlng_pb2,
+):
```

Comment:
The generator must not undo the change to stop overwriting the modules of shared message types (#5364).

Reply:
just fixed this in the tasks generation if you want to copy/pasta the same transformation (or at least it should be pretty similar)